validate/validate.go

```diff
@@ -337,11 +337,31 @@ func (v *Validator) CheckRlimits() (msgs []string) {
 func supportedMountTypes(OS string, hostSpecific bool) (map[string]bool, error) {
 	supportedTypes := make(map[string]bool)
 
-	if OS != "linux" && OS != "windows" {
+	if OS != "linux" && OS != "solaris" {
 		logrus.Warnf("%v is not supported to check mount type", OS)
```

Comment:
Repeating @Mashimiao's [earlier comment][1], since it still applies:

> Windows does not support type in mount, there is no need to output warning and we should accept this change after spec v1.0 released.

`type` is not defined for Windows since opencontainers/runtime-spec#854 (which you site in the comment opening this PR).  So configs which set it are taking advantage of the [extensibility rules][2] that allow them to do so.  Individual runtimes may or may not act upon the extention, but it's still a valid config.

There may be users who are interested in knowing if a runtime MUST support a given config (e.g. if that config can be portably used with any OCI-compliant runtime).  In that case, we would want to warn/error when we found extentions like this.  More thoughts on that sort of thing in opencontainers/image-tools#66.  However, neither runtime-tools nor image-tools currently provides switches for that sort of thing, so for the time being I'd rather silently ignore the `type` value (if any) in Windows configs.

[1]: https://github.com/opencontainers/runtime-tools/pull/397#discussion_r122376942
[2]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0/config.md#extensibility